src/wasm-lib/kcl/src/executor.rs

```diff
@@ -1695,6 +1721,21 @@ impl Path {
         }
     }
 
+    /// Where does this path segment start?
+    pub fn get_from(&self) -> &[f64; 2] {
+        &self.get_base().from
+    }
+    /// Where does this path segment end?
+    pub fn get_to(&self) -> &[f64; 2] {
+        &self.get_base().to
+    }
```

Comment:
Why the unnecessary indirection?

Reply:
In the future we're not going to know the start/end of lines so easily (we'll need to query wasm or the API) so this change is preparation for that.

Reply:
get_base() though is already abstracting